models/webhook_test.go

```diff
@@ -245,3 +245,39 @@ func TestUpdateHookTask(t *testing.T) {
 	assert.NoError(t, UpdateHookTask(hook))
 	AssertExistsAndLoadBean(t, hook)
 }
+
+func TestDeleteDeliveredHookTasks_DeletesDelivered(t *testing.T) {
+	assert.NoError(t, PrepareTestDatabase())
```

Comment:
can you insert 2 and limit to 1
and check if only the right one has been deleted?

Reply:
sure, I added a test where I created a new webhook that was delievered "now" and then called the delete method where 1 is kept...and then verified that the newly created webhook is still present.